gce/survival-training/README-tf-estimator.md

```diff
@@ -0,0 +1,404 @@
+
+To do so, we must create a source repo, which [you can do from Cloud Console here](https://console.cloud.google.com/code/develop/repo). Alternatively, you can use the `gcloud` tool:
+
+```bash
+gcloud source repos create tensorflow-models
+```
+
+If this is the first time you are using source repos, make sure to run the following command from your local machine:
+
+```bash
+git config credential.helper gcloud.sh
+```
+
+Now navigate to your local clone of [tensorflow/models](https://github.com/tensorflow/models) and add this new repo as a remote:
+
+```
+git remote add gcp https://source.developers.google.com/p/$(gcloud config get-value project)/r/tensorflow-models
+```
+
+Finally,
+
+```bash
+git push -u gcp-or-whatever master
+```
+
+This push might take some time because of the size of `tensorflow/models`.
+
+
+## Startup script
+
+We will use the [tf-estimator-startup.sh](./gce/tf-estimator-startup.sh) script, which is only a slight modification of [our origina dummy script](./gce/startup.sh).
+
+
+## Instance metadata
+
+We can set instance metadata either by editing our instance in the cloud console or through the `gcloud` CLI. Let us use `gcloud` for this.
+
+First, let us define two environment variables:
+
+```bash
+export DATA_DIR=gs://$(gcloud config get-value project)-cifar-10-data/ JOB_DIR=gs://$(gcloud config get-value project)-cifar-10-checkpoints
+```
+
+You should also export your username on the GCE instance (the one under which you created your environment) into the `GCE_USER` environment variable. In my case, I export
+
+```bash
+export GCE_USER=nkash
+```
+
+
+Assuming you are executing the command from the same directory as this guide:
+
+```bash
+gcloud compute instances add-metadata cifar10-estimator \
+    --metadata-from-file startup-script=./gce/tf-estimator-startup.sh \
+    --metadata gceUser=$GCE_USER,trainerRepo=tensorflow-models,trainerModule=tutorials.image.cifar10_estimator.cifar10_main,dataDir=$DATA_DIR,jobDir=$JOB_DIR,trainSteps=99999999,numGpus=4,momentum=0.9,weightDecay=0.0002,learningRate=0.1,batchNormDecay=0.997,batchNormEpsilon=0.00001
+```
+
+(Note: Remember that we had called our instances `cifar10-estimator`. If you name yours something different, you should make the appropriate modification to the above command.)
+
+You can verify that this metadata has actually been attached to the instance using:
+
+```bash
+gcloud compute instances describe cifar10-estimator
+```
+
+
+## Starting your training job
+
+We are now ready to start our training job. And starting it is very simple with the custom metadata and the startup script set on our GCE instance. All we do is:
+
+```bash
+gcloud compute instances start cifar10-estimator
```

Comment:
same here.  consider adding this into a script, maybe just call it `start_training.sh` or something obvious.